packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -79,6 +79,21 @@ export const handleException = async (exception: unknown): Promise<void> => {
   process.exit(1);
 };
 
+/**
+ * Handle rejected promises that weren't caught or awaited anywhere in the code.
+ */
+export const handleUnhandledRejection = (reason: Error | $TSAny): void => {
```

Comment:
Should we report to analytics here? Or is that handled elsewhere?

Reply:
This is going to analytics! Its handled in 'uncaughtException' node handler.